clang/lib/AST/Interp/ByteCodeExprGen.h

```diff
@@ -368,6 +369,7 @@ template <class Emitter> class LocalScope : public VariableScope<Emitter> {
       return;
     this->emitDestructors();
     this->Ctx->emitDestroy(*Idx, SourceInfo{});
+    removeStoredOpaqueValues();
```

Comment:
Inline this function here.

Reply:
This function has 2 callsites, you want the other callsite inlined too, or you want me to remove that?

Reply:
Ah, didn't see that. Well, it's fine to leave it then.